### DISCUSSION TITLE
Backfill BG data does not happen in AAPS

### DISCUSSION THREAD

**Discussion by dar-kod** [NightscoutFoundation/xDrip#2933] [General]
I am using AAPS 3.2.0 dev, G6 and the latest xDrip+ nightly. I have noticed that if I am away from my phone for more than 5 minutes, the backfill BG data does not happen in AAPS. My BG source is xDrip+. It worked before. I'm not sure when it started. Is it a problem with xDrip+?

**Comment by Navid200** [comment]
Is there any way you can test and identify which version of xDrip does and which version does not have this problem?

**Comment by dar-kod** [comment]
I think it started when this broadcasts of boluses from AAPS to xDrip+ was solved in xDrip+, but I am not sure anymore.  I'm not sure if I have time to test which version it started with.
